catalogs/catalog-lakehouse-iceberg/src/test/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/converter/TestBaseConvert.java

```diff
@@ -120,6 +133,59 @@ protected static Types.NestedField[] createNestedField(String... colNames) {
     return results.toArray(new Types.NestedField[0]);
   }
 
+  protected static String convertIcebergTransform(SortField sortField) {
```

Comment:
it's hard to understand without knowing what transform represents, could you add some description about the rules?

Reply:
updated.

Reply:
updated.